selfdrive/camerad/cameras/camera_common.cc

```diff
@@ -295,9 +295,7 @@ static void create_thumbnail(MultiCameraState *s, const CameraBuf *b) {
   thumbnaild.setTimestampEof(b->cur_frame_data.timestamp_eof);
   thumbnaild.setThumbnail(kj::arrayPtr((const uint8_t*)thumbnail_buffer, thumbnail_len));
 
-  if (s->pm != NULL) {
```

Comment:
Why is the null check no longer necessary? Is the pubmaster initialized on all camera types?

Reply:
yes,except for the camera_frame_stream that does not call `processing_thread`

Reply:
Shall we just leave the null check just for extra safety? Don't want to have unexpected surprises if we refactor the code later.

Reply:
how about check it before call publish_thumbnail

Reply:
that's fine too!